lib/Game.rb

```diff
@@ -0,0 +1,70 @@
+
+class Game
+    include UserInterface
+    def initialize(board, player1, player2)
+        @board =  board
+        @player_1 = player1
+        @player_2 = player2
+        @current_player = @player_1
+    end
+    WIN_COMBO = [
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+      [0, 3, 6],
+      [1, 4, 7],
+      [2, 5, 8],
+      [0, 4, 8],
+      [6, 4, 2]
+     ]
+
+    def play
+        loop do 
+            show_board(@board)
+            ask_move(@board , @current_player)
+            break if is_game_over?
+            @current_player = @current_player == @player_1 ? @player_2 : @player_1
+        end
+    end
+
+    def is_game_over?
+        if is_victory? == true
+            show_info(1)
```

Comment:
Can we simply return `true` or `false` for this method? Any method with `?` in the end is conventionally supposed to just return a boolean. When you test this method it will be very hard to make an assertion